ng-deploy/src/ng-deploy/project-prompt.ts

```diff
@@ -0,0 +1,40 @@
+import * as inquirer from 'inquirer';
+import { Project } from './types';
+const fuzzy = require('fuzzy');
+
+inquirer.registerPrompt('autocomplete', require('inquirer-autocomplete-prompt'));
+
+const isProject = (elem: Project | { original: Project }): elem is Project => {
```

Comment:
Leave a comment, on why project can be not a project?
Or maybe normalize them in advance?

Reply:
fuzzy passes either the original list of projects or an internal object which contains the project as a property. I'll add this as a comment.